.circleci/config.yml

```
@@ -0,0 +1,48 @@
+version: 2
+jobs:
+  build:
+    working_directory: /app
+    docker:
+      - image: integratedexperts/circleci2-builder
+        environment:
+          COMPOSER_ALLOW_SUPERUSER: 1
+          COMPOSE_PROJECT_NAME: govcms8
+    steps:
+      - checkout
+      - run:
+          name: Docker-compose validate
+          command: docker-compose config -q
+      - run:
+          name: Update codebase to follow CI configuration
+          command: |
+            sed -i -e "/###/d" docker-compose.yml
+            cp .env.default .env
+      - setup_remote_docker
+      - run:
+          name: Start amazeeio-network
+          command: docker network prune -f && docker network create amazeeio-network
```

```suggestion
          command: docker network prune -f && docker network create amazeeio-network || true
```
Add true option to allow local circleCI testing